brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RuntimeEnvironmentModifier.java

```diff
@@ -53,7 +53,7 @@ public void apply(EntitySpec<?> entitySpec, String nodeId, ToscaApplication tosc
             }
 
             // We know that formatString will return a BrooklynDslDeferredSupplier as the 2nd argument is not yet resolved
-            BrooklynDslDeferredSupplier deferredRunDir = (BrooklynDslDeferredSupplier) BrooklynDslCommon.formatString("%s/%s", BrooklynDslCommon.attributeWhenReady("run.dir"), artifactId);
+            BrooklynDslDeferredSupplier deferredRunDir = (BrooklynDslDeferredSupplier) BrooklynDslCommon.formatString("%s/%s", BrooklynDslCommon.attributeWhenReady("install.dir"), artifactId);
```

Comment:
I think this is incorrect - we shouldn't be placing files for variants of the same blueprint in the same directory. Can we add a pre-install command to create `run.dir` earlier than would otherwise happen?